worldcup/scripts/add_auction.py

```diff
@@ -3,12 +3,12 @@
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
-    parser.add_argument("cup", help="cup name")
-    parser.add_argument("gambler", help="gamber name")
+    parser.add_argument("db", help="db name")
+    parser.add_argument("gambler", help="gambler name")
     parser.add_argument("team", help="team name")
-    parser.add_argument("price", help="auction price")
+    parser.add_argument("price", help="bid price")
     args = parser.parse_args()
 
-    insert_auction(args.cup, args.team, args.gambler, args.price)
+    insert_auction(args.team, args.gambler, args.price, dbname=args.db)
```

Comment:
这样有没有更优雅和 consistent 一点？

```suggestion
    with app.app_context():
        g.dbname = args.db
        insert_auction(team=args.team, gambler=args.gambler, args.price)
```

Reply:
add_auction里要import app么？

Reply:
引用 tournamentdb 就需要 app_context 吧

Reply:
那个是调用insert_auction的时候在model里调用的

Reply:
这个是为了把 insert_auction() 的 dbname 参数给干掉，丑，而且跟其他接口不一致